app/models/host.rb

```diff
@@ -59,6 +59,17 @@ def readonly?
     true
   end
 
+  def self.os_version_query(path, values, include = true)
+    query = include ? :in : :not_in
+    path = %i[major minor].include?(path) && path
+
+    Arel::Nodes::NamedFunction.new('CAST', [Arel::Nodes::InfixOperation.new(
+      '->',
```

Comment:
Interesting that `->` is changed to `@>` for the query.

Reply:
`@>` is for partial matching

Reply:
My bad, I forgot to check out the PR's code. :-(